.travis.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+set -ex
+hhvm --version
+apt-get update -y
+apt-get install -y wget curl git
+curl https://getcomposer.org/installer | hhvm -d hhvm.jit=0 --php -- /dev/stdin --install-dir=/usr/local/bin --filename=composer
```

Comment:
It isn't; I'm not keen on changing this until we can happily rebuild the images on a much more frequent basis than the HHVM release schedule.